sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/ProtobufDataImporterTest.java

```diff
@@ -68,26 +69,32 @@ public void prepare() throws Exception {
     tester = SensorContextTester.create(new File("src/test/resources"));
     tester.fileSystem().setWorkDir(workDir);
 
-    inputFile = new TestInputFileBuilder(tester.module().key(), "Program.cs")
+    firstFile = new TestInputFileBuilder(tester.module().key(), "Program.cs")
       .setLanguage("cs")
       .initMetadata(new String(Files.readAllBytes(csFile), StandardCharsets.UTF_8))
       .build();
-    tester.fileSystem().add(inputFile);
+    tester.fileSystem().add(firstFile);
 
-    when(fileLinesContextFactory.createFor(inputFile)).thenReturn(fileLinesContext);
+    secondFile = new TestInputFileBuilder(tester.module().key(), "Data.cs")
+      .setLanguage("cs")
+      .initMetadata(new String(Files.readAllBytes(csFile), StandardCharsets.UTF_8))
+      .build();
+    tester.fileSystem().add(firstFile);
+
+    when(fileLinesContextFactory.createFor(firstFile)).thenReturn(fileLinesContext);
```

Comment:
shouldn't you do lines 84 and 88 for `secondFile` as well?

Reply:
Line 88: It's good idea to move the scaffolding here to simplify the test below. No harm done to other tests.

Line 84: This is now needed after new line 90 was added.